src/table/components/virtualized-table/__tests__/TableContainer.spec.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
Had to use `@testing-library/user-event` to get the proper chain of events triggerd by a click